### ISSUE TITLE
[BUG] Content under layout is not animated

### ISSUE THREAD

**Issue by vavra7** [framer/motion#1124]
It seems that applying layout prop animates everything within element but not parents height. This restricts usage of layout everywhere where element follows another content. Content under such an element then jumps on change of the layout element.

In real world application I found prop layout useful for example in notifications because you can guarantee that there si no content under. Unfortunately in most cases of custom components you expect content around and therefore layout can not be used.

Please check example:
https://codesandbox.io/s/framer-motion-content-under-layout-9kw5g

**Comment by mattgperry** [comment]
Yeah you’d have to add layout to any surrounding content you wish to animate

**Comment by vavra7** [comment]
That is not possible because when you create component for use by third party user the user has only the finished component and has nothing to do with animation library that was used for the component. Therefore he can not apply "layout" on elements around the component.

**Comment by mattgperry** [comment]
Ah yes I see what you mean. I’m afraid this is impossible to fix in a
performant manner with layout animations. You could animate height instead?

Op za 1 mei 2021 om 12:25 schreef vavra7 ***@***.***>

**Comment by vavra7** [comment]
Yeah it seems that animating height do the trick. Alright then. Thanks for your answer.
